docs/usage.md

```diff
@@ -6,58 +6,157 @@
+
+- [CLUSTALO](http://clustal.org/omega/#Documentation) (accepts guide tree)
+- [FAMSA](https://github.com/refresh-bio/FAMSA) (accepts guide tree)
+- [KALIGN](https://github.com/TimoLassmann/kalign)
+- [LEARNMSA](https://github.com/Gaius-Augustus/learnMSA)
+- [MAFFT](https://mafft.cbrc.jp/alignment/server/index.html)
+- [MAGUS](https://github.com/vlasmirnov/MAGUS) (accepts guide tree)
+- [MUSCLE5](https://drive5.com/muscle5/manual/)
+- [TCOFFEE](https://tcoffee.readthedocs.io/en/latest/index.html) (accepts guide tree)
+
+**SEQUENCE- and STRUCTURE-BASED** (require both fasta and structures as input):
+
+- [3DCOFFEE](https://tcoffee.org/Projects/expresso/index.html) (accepts guide tree)
+
+**STRUCTURE-BASED** (only require stuctures as input):
+
+- [MTMALIGN](https://bio.tools/mtm-align)
+
+### 4. EVALUATE
+
+Optionally, the produced MSAs can be evaluated. Skip with `--skip_eval`.
+
+**SEQUENCE-BASED** (no extra input required):
+
+1. Calculate number of gaps and its average across sequences. Activate using `--calc_gaps` (default: true).
+
+**REFERENCE-BASED**:
+
+The reference MSAs (see samplesheet) are be used to evaluate the quality of the produced MSA.
+
+2. Sum Of Pairs. Calculates the SP score using the [TCOFFEE](https://tcoffee.readthedocs.io/en/latest/index.html) implementation. Activate using `--calc_sp` (default: true).
+3. Total column. Calculates the TC score [TCOFFEE](https://tcoffee.readthedocs.io/en/latest/index.html). Activate using `--calc_tc` (default: true).
+
+**STRUCTURE-BASED**:
+
+The provided structures (see samplesheet) are used to evaluate the quality of the alignment.
+
+4. iRMSD. Calculates the iRMSD using the [TCOFFEE](https://tcoffee.readthedocs.io/en/latest/index.html) implementation. Activate using `--calc_irmsd` (default: false).
+
+### 5. REPORT
+
+Finally, a summary table with all the computed statistics and evaluations is reported in MultiQC (Skip by `--skip_multiqc`).
+Moreover, a shiny app is prepared with interactive summary plots.
+
+> [!WARNING]
+> You will need to have [shiny](https://shiny.posit.co/py/) installed to run it! See [output documentation](https://nf-co.re/multiplesequencealign/output) for more infos.
 
 ## Samplesheet input
 
-You will need to create a samplesheet with information about the samples you would like to analyse before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row as shown in the examples below.
+The sample sheet defines the input data that the pipeline will process.
+It should look like this:
 
-```bash
---input '[path to samplesheet file]'
+`samplesheet.csv`:
+
+```csv
+id,fasta,reference,structures
+seatoxin,seatoxin.fa,seatoxin-ref.fa,seatoxin_structures
+toxin,toxin.fa,toxin-ref.fa,toxin_structures
 ```
 
-### Multiple runs of the same sample
+Each row represents a set of sequences (in this case the seatoxin and toxin protein families) to be processed.
```

Comment:
this is not generic? I mean the example is still for "seatoxin and toxin protein families"?

Reply:
or maybe I am missing something, before we have two examples and now only one, right?

Reply:
sisi it is for 2 families, like in the main readme, but now we go into details on what each column is etc